src/Microsoft.DotNet.Configurer/NuGetPackagesArchiver.cs

```diff
@@ -10,33 +10,15 @@ namespace Microsoft.DotNet.Configurer
 {
     public class NuGetPackagesArchiver : INuGetPackagesArchiver
     {
-        private ITemporaryDirectory _temporaryDirectory;
-
         public string NuGetPackagesArchive =>
             Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, "nuGetPackagesArchive.lzma"));
 
-        public NuGetPackagesArchiver() : this(FileSystemWrapper.Default.Directory)
-        {
-        }
-
-        internal NuGetPackagesArchiver(IDirectory directory)
-        {
-            _temporaryDirectory = directory.CreateTemporaryDirectory();
-        }
-
-        public string ExtractArchive()
+        public void ExtractArchive(string archiveDestination)
```

Comment:
Lots of public API changes. I know I've seen other PRs where changing Microsoft.DotNet.* public API was turned away. How can I tell which assemblies are OK to break like this?